docs/sources/administration/correlations/create-a-new-correlation/index.md

```diff
@@ -13,6 +13,25 @@ weight: 40
 
 Make sure you have permission to add new correlations. Only users with write permissions to data sources can define new correlations.
 
+## Create a correlation in Explore's Correlations Editor
+
+1. Go to Explore page.
+1. Select a data source that you would like to be [the source data source]({{< relref "/docs/grafana/latest/administration/correlations/correlation-configuration#source-data-source-and-result-field" >}}) for a new correlation.
+1. Run a query producing data in [a supported visualization]({{< relref "/docs/grafana/latest/administration/correlations/#correlations" >}}).
+1. Click "+ Add" button in the top toolbar and select "Add correlation" (you can also select "Correlations Editor" from the [Command Palette]({{< relref "/docs/grafana/latest/search/#command-palette" >}})).
+1. Explore is now in Correlations Editor mode indicated by a blue border and top bar. You can exit Correlations Editor using the Exit button in the top bar.
+1. The visualization is enriched with links allowing to create new correlations. Links are attached to the data that you can use to build a new query:
+   - Logs: links are displayed next to field values inside log details for each log row
+   - Table: every table cell is a link
+1. Click on a link to begin adding new correlation. Links are associated with a field that will be used as [a results field of a correlation]({{< relref "/docs/grafana/latest/administration/correlations/correlation-configuration" >}}).
+1. Explore opens a split view. Use the right pane to setup [the target query source of the correlation]({{< relref "/docs/grafana/latest/administration/correlations/correlation-configuration#target-query" >}}).
+1. Build target query using [variables syntax]({{< relref "/docs/grafana/latest/dashboards/variables/variable-syntax" >}}) with variables from the list provided at the top of the pane. The list contains sample values from the selected data row.
+1. Provide optional label and description. Label is used as the name of the link inside the visualization. I can contain variables.
```

Comment:
> I can contain variables.
🤯 I think you mean "It can contain variables" but I'm not sure we interpolate the label?

Reply:
Oh yes we do! It's useful to have more customized label on the button, like "show metrics for srv001 host"

Reply:
I can contain variables too :P updated in 874bf9a86edc116ca976bab5fa042ece8886e172

Reply:
🤯 again! Thanks for letting me know and including that in the docs